### PR TITLE
Tests for doSimpleXMLAssignment.

### PR DESCRIPTION
**Pull request by damonkelley**
Completes #27

### CODE REVIEW

**Comment by damonkelley** [comment]
ping @ldko @somexpert

**Comment by damonkelley** [comment]
I am not sure why, but Travis is claiming that the push build failed. If you check the details, you will see that in fact the build succeeded

**Comment by somexpert** [comment]
Should we also have a test where there's a nested element but chain does not list the parent? Not a leading question, really not sure.

File: tests/test_presentation.py [hunk 1]
```diff
@@ -573,3 +573,104 @@ def test_agent_type(self):
...
+        tree = etree.fromstring(
+            """<root>
+                <value>1</value>
+            </root>
+            """
+        )
+
+        chain = ['value']
+
+        # Since the function mutates the the object, we will verify that the
```
Comment (somexpert) [inline tests/test_presentation.py]: `the the`

File: tests/test_presentation.py [hunk 2]
```diff
@@ -573,3 +573,104 @@ def test_agent_type(self):
...
+
+class TestDoSimpleXMLAssignment:
+
+    @pytest.fixture
+    def record_obj(self):
+        class Record(object):
+            value = None
+        return Record()
+
+    def test_attribute_not_set_when_element_value_is_none(self, record_obj):
```
Comment (somexpert) [inline tests/test_presentation.py]: Could we rename this test to indicate the root (fieldname not matching element name) of the test?

**Comment by somexpert** [comment]
:+1:

**Comment by ldko** [comment]
:+1:

**Pull request merged**